easybuild/easyconfigs/s/sympy/sympy-1.3-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'PythonPackage'
+
+name = 'sympy'
+version = '1.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://sympy.org/'
+description = """
+ SymPy is a Python library for symbolic mathematics. It aims to
+ become a full-featured computer algebra system (CAS) while keeping the code as
+ simple as possible in order to be comprehensible and easily extensible. SymPy
+ is written entirely in Python and does not require any external libraries.
+"""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+patches = ['sympy-1.3_symlink.patch']
```

Comment:
@jose-d Please use `%(version)s` rather than hardcoding the version here, since they match:

```python
patches = ['sympy-%(version)s_symlink.patch']